### DISCUSSION TITLE
Github Copilot Chat

### DISCUSSION THREAD

**Discussion by Fluffymuzzy** [community/community#60880] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

how long to wait for a copilot chat application to be processed?

**Comment by pog5** [comment]
It should be processed as fast as possible (5-10 minutes MAX)

**Reply by Fluffymuzzy** [reply]
I've been waiting for half a day with this message: "Thanks for joining!
You’re already on the waitlist for GitHub Copilot chat private beta!
We’ll email you when we’ve enabled it on your account.
"

nothing came in the mail.

**Reply by Fluffymuzzy** [reply]
To use chat you have to wait for this message? Because when I install the extension in vscode, it doesn't show up in the panel. I tried both in regular vscode and in insaders.

**Reply by marcbunyan** [reply]
...can you confirm where that waiting time is documented?

**Comment by marcbunyan** [comment]
> It should be processed as fast as possible (5-10 minutes MAX)

**Comment by davecheney** [comment, marked as answer]
@Fluffymuzzy i've confirmed you have been inviteded.

**Comment by naftalysh** [comment]
I can't seem to find how to cancel my copilot payments
![image](https://github.com/community/community/assets/640645/35dac6f5-154b-49d1-91df-d48889f74eef)

**Reply by davecheney** [reply]
@naftalysh this discussion is closed. please contact support for assistance with billing.

**Reply by naftalysh** [reply]
what is the contact email?

**Reply by davecheney** [reply]
https://support.github.com/contact
